examples/bq-log-alerting/versions.tf

```diff
@@ -15,5 +15,9 @@
  */
 
 terraform {
-  required_version = ">=0.12.6, <0.14"
+  required_version = ">=0.12.6"
```

Comment:
In the README it indicates that Terraform 13+ is required. modules/bq-log-alerting/README.md:162

Reply:
@aaronsutton updated to `0.13.6`

Reply:
It looks like we can't use a newer version
```
The configuration is valid.
terraform_validate ./examples/bq-log-alerting 
Error: 
Unsupported Terraform Core version
 on versions.tf line 18, in terraform:
 18:   
required_version = ">=0.13.6"
This configuration does not support Terraform version 0.13.5. To proceed,
either choose another supported Terraform version or update this version
constraint. Version constraints are normally set for good reason, so updating
the constraint may lead to other errors or unexpected behavior.
```